manifests/job/present.pp

```diff
@@ -59,6 +60,7 @@
       content => $d,
       require => Exec['jenkins-cli'],
     }
+    $require = File[$tmp_config_path]
```

Comment:
Alternative is to add `before => Exec["jenkins create-job ${jobname}"]` on the file resource since the exec is there unconditionally